Re-add Cook municipality to vw_pin_universe pipeline (#863)
ccao-data/data-architecture: dbt/models/location/columns.md

2-digit Chicago police district number
{% enddocs %}

## combined_municipality_name

{% docs column_combined_municipality_name %}
Combines `tax_municipality_name` and `cook_municipality_name`.
`tax_municipality_name` has both NULL values and empty array values.

> If the value is NULL, we replace it with `cook_municipality_name`. If the value 

Author: I don't think there are any NULL values for combined_municipality_name. The 'the' refers to `tax_municipality_name`, here which I'll just clarify.

select * from "z_ci_862_re_add_spatialmunicipality_data_location"."vw_pin10_location"
where combined_municipality_name is NULL

2-digit Chicago police district number
{% enddocs %}

> ## combined_municipality

Reviewer: ```suggestion
## combined_municipality_name
```

> {% docs column_combined_municipality %}

Reviewer: ```suggestion
{% docs column_combined_municipality_name %}
```

> This is a column which combines tax_municipality_name and cook_municipality_name.
> This is because tax_municipality_name does not have values for Cicero and 
> for some new parcels. 
> Steps:
> 1: We take the value of tax_municipality_name if
> the cardinality is >0 (non-NULL).
> 2: We check if the PIN is in Cicero in cook_municipality_name. There are two
> names for this Village and Town of Cicero. These switch over completely in 2007.
> Because of this, we coalesce with either of these values.
> 3: We check if the PIN is new in iasworld.pardat. If it is, we coalesce the 
> value from cook_municipality_name. We then run these values through a
> crosswalk file to make sure they align with tax_municipality_name.
> 4: We return the remaining NULL values from tax_municipality_name

Reviewer: ```suggestion
Combines `tax_municipality_name` and `cook_municipality_name`.
While `tax_municipality_name` is more dependable, it does not have values for
Cicero due to its status as both a township and a municipality, or parcels that
have been created more recently than our most recent tax data.
Steps:
1. Use the value of `tax_municipality_name` if it contains one or more non-null
values.
2. Check if the PIN has a value of "Village of Cicero" or "Town of Cicero" for
`cook_municipality_name` depending on whether the data is pre- or post-2007.
3. If the PIN is newer than the most recent year of tax data coalesce the 
value from `cook_municipality_name`. We then run these values through a
crosswalk file to make sure they align semantically with
`tax_municipality_name`.
4. Return the remaining NULL values from `tax_municipality_name`
```

2-digit Cook County judicial district number
{% enddocs %}

## cook_municipality_name

{% docs column_cook_municipality_name %}

> Cook county municipality name from Cook County GIS files

Reviewer: ```suggestion
Municipality name according to Cook County  GIS geodatabase files.
```

Reviewer: Not sure the period at the end of my suggestion should be there, sorry.

{% enddocs %}

## cook_municipality_num

{% docs column_cook_municipality_num %}

> Cook county municipality number. This does not correspond with tax_municipality_num

Reviewer: ```suggestion
Municipality number according to Cook County  GIS geodatabase files. Does not correspond with `tax_municipality_num`
```
